spec/acceptance/help_spec.rb

```diff
@@ -1,26 +1,47 @@
 require 'spec_helper_acceptance'
 
-describe "C98923 - Verify that 'puppet certregen --help' prints help text" do
-    describe command("puppet certregen --help") do
-      its(:stdout) { should match( /.*USAGE: puppet certregen <action>.*/ ) }
-      its(:stdout) { should match( /.*See 'puppet man certregen' or 'man puppet-certregen' for full help.*/ ) }
+describe "pupper help certregen" do
+  # NOTE: MODULES-4733 certregen is not currently compatible with ruby < 1.9
+  ruby_ver = 0
```

Comment:
Thanks @adrienthebo . There is a Beaker helper that can simplify these checks. I have refactored the code to utilize it.